pages/video-download/video-download.view.scss

```diff
@@ -162,3 +162,10 @@ a:hover {
 		text-align: center;
 	}
 }
+
+.code-error {
+	padding: 8px;
+	display: inline-block;
+	width: 50%;
+	color: $error;
```

Comment:
sass doesnt find $error

Reply:
Ups! My fail. That variable was defined in #27 T_T We could mege this into #27, and that one into dev, or something like that

Reply:
#27 is now conflicting with dev after merging #28 should be updated first, and some review would be desirable, maybe @DevStarlight could help